apps/cli/src/service-container/service-container.ts

```diff
@@ -777,15 +777,15 @@ export class ServiceContainer {
     const userId = (await this.stateService.getUserId()) as UserId;
     await Promise.all([
       this.eventUploadService.uploadEvents(userId as UserId),
-      this.cryptoService.clearKeys(),
+      this.cryptoService.clearKeys(userId),
       this.cipherService.clear(userId),
       this.folderService.clear(userId),
       this.collectionService.clear(userId as UserId),
     ]);
 
     await this.stateEventRunnerService.handleEvent("logout", userId);
 
-    await this.stateService.clean();
+    await this.stateService.clean({ userId: userId });
```

Comment:
suggestion(non-blocking): simplify
```suggestion
    await this.stateService.clean({ userId });
```